src/app/components/home/home.component.html

```diff
@@ -12,39 +12,12 @@ <h1 class="display-2" style="font-weight: bold">ברוכים הבאים 🖖</h1
         עיינו בתכנים
       </button>
     </div>
-
   </div>
-  <ol class="carousel-indicators">
-    <li data-target="#carouselIndicatorsStandard" data-slide-to="0" class="active"></li>
-    <li data-target="#carouselIndicatorsStandard" data-slide-to="1"></li>
-    <li data-target="#carouselIndicatorsStandard" data-slide-to="2"></li>
-  </ol>
+
   <div class="carousel-inner shadow-lg">
     <div class="carousel-item active">
       <div class="background-img" style="background-image: url('assets/imgs/377703-PBLQL4-228.jpg');"></div>
-      <div class="carousel-caption d-none d-md-block">
-
-      </div>
-    </div>
-    <div class="carousel-item">
-      <div class="background-img" style="background-image: url('assets/imgs/377703-PBLQL4-228.jpg');"></div>
-      <div class="carousel-caption d-none d-md-block">
-
-      </div>
-    </div>
-    <div class="carousel-item">
-      <div class="background-img" style="background-image: url('assets/imgs/377703-PBLQL4-228.jpg');"></div>
-      <div class="carousel-caption d-none d-md-block">
-
-      </div>
+      <div class="carousel-caption d-none d-md-block"></div>
```

Comment:
You left an empty carousel here. Does it contain anything I don't see, or it can be removed totally?